test/unit/Constellation/OperatorDistributor/test-onEthBeaconRewardsReceived.ts

```diff
@@ -0,0 +1,153 @@
+                        ethers.utils.parseEther("1"),
+                        ethers.utils.parseEther("1"),
+                        ethers.utils.parseEther("1"),
+                    )).to.be.revertedWith("Transfer to treasury failed");
+
+                    expect(await ethers.provider.getBalance(treasury.address)).to.equal(ethers.BigNumber.from("0"));
+                    expect(await ethers.provider.getBalance(operatorReward.address)).to.equal(ethers.BigNumber.from("0"));
+                    expect(await operatorDistributor.oracleError()).to.equal(ethers.BigNumber.from("0"));
+                });
+            });
+            describe("when the transfer to node operator fails", function () {
+                it("reverts", async function () {
+                    await expect(treasury.setRejectCall(false)).to.not.be.reverted;
+                    await expect(operatorReward.setRejectCall(true)).to.not.be.reverted;
+
+                    expect(await ethers.provider.getBalance(treasury.address)).to.equal(ethers.BigNumber.from("0"));
+                    expect(await ethers.provider.getBalance(operatorReward.address)).to.equal(ethers.BigNumber.from("0"));
+
+                    await expect(operatorDistributor.connect(owner).onEthBeaconRewardsReceived(
+                        ethers.utils.parseEther("1"),
+                        ethers.utils.parseEther("1"),
+                        ethers.utils.parseEther("1"),
+                    )).to.be.revertedWith("Transfer to operator fee address failed");
+
+                    expect(await ethers.provider.getBalance(treasury.address)).to.equal(ethers.BigNumber.from("0"));
+                    expect(await ethers.provider.getBalance(operatorReward.address)).to.equal(ethers.BigNumber.from("0"));
+                    expect(await operatorDistributor.oracleError()).to.equal(ethers.BigNumber.from("0"));
+
+                });
+            });
+            describe("when both transfer succeeds", function () {
+                it("increases the oracleError amount", async function () {
+                    await expect(treasury.setRejectCall(false)).to.not.be.reverted;
+                    await expect(operatorReward.setRejectCall(false)).to.not.be.reverted;
+
+                    expect(await ethers.provider.getBalance(treasury.address)).to.equal(ethers.BigNumber.from("0"));
+                    expect(await ethers.provider.getBalance(operatorReward.address)).to.equal(ethers.BigNumber.from("0"));
+
+                    await expect(operatorDistributor.connect(owner).onEthBeaconRewardsReceived(
+                        ethers.utils.parseEther("10"),
+                        ethers.utils.parseEther("0.1"),
+                        ethers.utils.parseEther("0.2"),
+                    )).to.not.be.rejected;
+
+                    expect(await ethers.provider.getBalance(treasury.address)).to.equal(ethers.BigNumber.from("1000000000000000000"));
```

Comment:
FYI you can use the following for something a little more readable
```suggestion
                    expect(await ethers.provider.getBalance(treasury.address)).to.equal(ethers.utils.parseEther("1"));
```